packages/contentstack-auth/package.json

```diff
@@ -21,16 +21,13 @@
   "dependencies": {
     "@contentstack/cli-command": "^1.0.0",
     "@contentstack/cli-utilities": "^1.0.0",
-    "@oclif/command": "^1.8.16",
-    "@oclif/config": "^1.18.3",
     "chalk": "^4.0.0",
     "debug": "^4.1.1",
     "inquirer": "^7.1.0",
     "winston": "^3.7.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/plugin-help": "^5.1.12",
+    "oclif": "^3",
```

Comment:
Can we use the latest oclif verion [3.1.2] ?

Reply:
done